### PR TITLE
Alcohol symptom runtime fix

### PR DESCRIPTION
**Pull request by Job-IDev**
## About The Pull Request

There is currently a runtime error when someone is getting the Autobrewery Syndrome symptom.
The symptom itself only has 1 severity, any virus that doesn't go above 3 severity will make this runtime error happen.

Basically what is happening:

It's checking A.stage + severity.
If it's 4 to 5, then it will say x
If it's 6 to 7, then it will say y
If it's 8 to Infinite then it will say z

But during the start, the sum will be 2. Which isn't in the switch.
This is where the change comes in.

(Note that `-INFINITY` does not work and crashes the compiler on boot)

## Why It's Good For The Game

Just another removal of a runtime error.

## Testing Photographs and Procedure

During the testing of symptoms, I noticed a runtime error.
I found the problem. After implementing this it works as intended.

## Changelog
:cl:
fix: Fixed a runtime I've found located in the alcohol.dm file
/:cl:

### CODE REVIEW

File: code/datums/diseases/advance/symptoms/alcohol.dm [hunk 1]
```diff
@@ -35,7 +35,7 @@
 	var/mob/living/carbon/M = A.affected_mob
 	var/list/warningstrings = list()
 	switch(A.stage + severity)
-		if(4 to 5)
+		if(-5 to 5)
```
Comment (Archanial) [inline code/datums/diseases/advance/symptoms/alcohol.dm]: This is equal to default and doesn't need arbitrary -5 value to catch all other cases

**Review by Archanial — CHANGES_REQUESTED** [review]

File: code/datums/diseases/advance/symptoms/alcohol.dm (hunk 1 above)
Comment (Job-IDev) [reply code/datums/diseases/advance/symptoms/alcohol.dm]: It doesn't need to be `-5` and I'd rather use `-INFINITY`, but unfortunately, it won't compile with that.
As I've explained in the PR it needs to include the negative values as well.

Comment (itsmeow) [reply code/datums/diseases/advance/symptoms/alcohol.dm]: Isn't there a NEGATIVE_INFINITY define

Comment (EvilDragonfiend) [reply code/datums/diseases/advance/symptoms/alcohol.dm]: `-INFINITY` is a thing. that's weird.

Comment (Archanial) [reply code/datums/diseases/advance/symptoms/alcohol.dm]: Default (in dm else) will catch any value from -infinity to 5

**Review by PowerfulBacon — APPROVED** [review]

**Review by Archanial — APPROVED** [review]

**Pull request merged**